frontend/src/metabase/components/Item.jsx

```diff
@@ -0,0 +1,44 @@
+/* eslint "react/prop-types": "warn" */
```

Comment:
Any suggestions in mind? I can't seem to think of anything better than maybe `ListItem`?